src/azure-account.ts

```diff
@@ -773,17 +785,47 @@ async function getEnvironments(): Promise<Environment[]> {
 
 	const config = workspace.getConfiguration('azure');
 	const ppe = config.get<Environment>('ppe');
-	if (ppe) {
-		return [
-			...staticEnvironments,
-			{
-				...ppe,
-				name: azurePPE
-			}
-		]
-	} else {
-		return staticEnvironments;
+	const apiProfile = config.get<Boolean>('target_azurestack_api_profile');
+	try {
```

Comment:
I know I suggested using a try-catch here, but since it just re-throws the error, I think it isn't needed. Since you check this `target_azurestack_api_profile` property first, others who are using the `ppe` setting for testing won't be impacted